helm/aws-pod-identity-webhook/templates/PodSecurityPolicy.yaml

```diff
@@ -3,6 +3,8 @@ kind: PodSecurityPolicy
 metadata:
   name: {{ .Values.name }}
   namespace: {{ .Values.namespace }}
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'runtime/default'
```

Comment:
why do you hardcode this if you added it to the values file?

Reply:
If I'm not mistaken, the way the seccompProfile is set in securitycontexts (RuntimeDefault) won't get accepted as annotation by PSPs ('runtime/default'). 
I believe this to be a remnant from a time that seccompprofiles were not yet able to be set directly into securityContext. 
But I'll have to fact check this.